src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -368,11 +368,15 @@ other Axom developers to use during development, in Axom GitLab CI testing, etc.
 
    Run the corresponding command for the system you are on::
 
-     # blueos
+     # blueos_3_ppc64le_ib_p9 (default cmake is 3.14, need >=3.21 for using-with-cmake example)
```

Comment:
I don't know if it's important to mention here, but RAJA, Umppire, etc. require cmake minimum version 3.23. For example, https://github.com/LLNL/RAJA/blob/develop/CMakeLists.txt#L47

Should Axom require the same min version when RAJA and Umpire are enabled?

Reply:
Axom currently varies the CMake requirement based on HIP/CUDA configuration:
https://github.com/LLNL/axom/blob/c88c85946ec7c7608484fe149db06a0c7beeaea5/src/CMakeLists.txt#L9-L15

Not sure what the potential consequences are of moving the new minimum from as old as `3.14` to `3.23`.